_examples/bundler/docker-entrypoint

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+require 'open3'
+
+# non-buffered stdout
+STDOUT.sync = true
+
+GEMFILE = File.read("/data/params/Gemfile").strip
```

Comment:
Since we're in data, the `/data/` part isn't needed.

Reply:
fixed